src/stories/Blocks/reservation/reservation.scss

```diff
@@ -0,0 +1,104 @@
+.reservation {
+  background-color: #f6f5f0;
+
+  // ReservationHeader
+  &__header {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: $s-md;
+    @include breakpoint-s {
+      flex-direction: row;
+    }
+
+    &__container {
+      margin-top: $s-lg;
+      @include breakpoint-s {
+        margin-left: 42px;
+      }
+    }
+
+    &__tag {
+      @extend .text-tags;
+      @extend .pl-16;
+      @extend .pr-16;
+      @extend .pt-4;
+      @extend .pb-4;
+      border: 1px solid $c-global-tertiary-1;
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    &__title {
+      @extend .text-header-h2;
+      margin-top: 22px;
+      @include breakpoint-s {
+        margin-bottom: 0;
+      }
+    }
+
+    &__author {
+      @extend .text-body-medium-regular;
+      margin-top: $s-sm;
+    }
+  }
+  // ReservationForm
+  &__form {
+    &__submit {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      @extend .mb-24;
+
+      @include breakpoint-s {
+        justify-content: end;
+        flex-direction: row;
+      }
+
+      &__info {
+        @extend .mb-16;
+        @extend .text-small-caption;
+        @include breakpoint-s {
+          margin-right: $s-lg;
+          margin-bottom: 0;
+        }
+      }
+    }
+
+    &__list {
+      display: grid;
+      gap: $s-sm;
+
+      &__item {
```

Comment:
Consider adding a hover shadow on the card like we do on the search result page. Visually it's the same card (just with different content) so it would make sense for it to behave the same, right?

Reply:
This does not appear in the design. and they are not links in the same way as search result items